core/designsystem/src/main/kotlin/com/dhc/designsystem/calendar/DhcCalendarController.kt

```diff
@@ -0,0 +1,30 @@
+package com.dhc.designsystem.calendar
+
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.Stable
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.saveable.rememberSaveable
+import java.time.LocalDate
+
+@Stable
+class DhcCalendarController(
+    private val initialDate: LocalDate = LocalDate.now(),
+    private val initialPage: Int = Int.MAX_VALUE / 2,
+) {
+    val currentDate = mutableStateOf(initialDate)
+
+    fun getDate(page: Int) = initialDate.plusDays(page.toLong() - initialPage)
```

Comment:
```suggestion
    fun getDate(page: Int): LocalDate = initialDate.plusDays(page.toLong() - initialPage)
```
Nit : 외부에서도 사용해야하는 공통컴포넌트 로직은 반환형을 붙여주는게 좋다는 것을 이펙티브 코틀린에서 봤던 것 같은데,, 반환형 추가해보는건 어떨까?

Reply:
@JeonK1 
음 나이스한 꿀팁인데 ??